tests/utils/log/test_secrets_masker.py

```diff
@@ -72,22 +72,6 @@ def test_args(self, logger, caplog):
 
         assert caplog.text == "INFO Cannot connect to user:***\n"
 
-    def test_non_redactable(self, logger, caplog):
```

Comment:
Could not find a way to trigger Exception :)

Reply:
Inherit from list or dict should trigger it, but don't mind deleting this test